decidim-admin/app/views/decidim/admin/participatory_processes/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<% content_for :title do %>
+  <h2><%= t "decidim.admin.titles.participatory_processes" %></h2>
+<% end %>
+
+<div class="actions title">
+  <%= link_to t("actions.new", scope: "decidim.admin", name: t("models.participatory_process.name", scope: "decidim.admin")), ['new', 'participatory_process'], class: 'new' %>
```

Comment:
Also, Rails has a `link_to_if` method (that I wasn't able to make it work, don't know why)